arch/arm/boot/dts/bcm2708_common.dtsi

```diff
@@ -132,6 +134,11 @@
 
 			interrupt-controller;
 			#interrupt-cells = <2>;
+
+			i2c0_gpio28: i2c0_gpio28 {
```

Comment:
It bothers me that this node isn't referenced anywhere. Beyond that, We would have to check that the firmware isn't going to dynamically reconfigure the function mapping on these pins, since i2c0 is a.k.a. i2c_vc.